ndbench-core/src/main/java/com/netflix/ndbench/core/config/IConfiguration.java

```diff
@@ -89,7 +96,25 @@ default int getNumReaders() {
     @DefaultValue("100")
     int getReadRateLimit();
 
+
     @DefaultValue("100")
+    @PropertyName(name=WRITE_RATE_LIMIT)
     int getWriteRateLimit();
 
+
+    @DefaultValue("false")
+    boolean isAutoTuneEnabled();
+
+
+    @DefaultValue("60")
```

Comment:
I have not read the new code changes, but I am not following why these 3 FPs are used in the internal nfndbench only and are not fully integrated with the OSS version? Moreover, can add me as a reviewer in the Stash PR so that I can have full context?